src/components/victory-candlestick/victory-candlestick.js

```diff
@@ -12,22 +12,24 @@ const fallbackProps = {
     height: 300,
     width: 450,
     candleColors: {
-      positive: "green",
-      negative: "red"
+      positive: "#ffffff",
+      negative: "#252525"
     }
   },
   style: {
     data: {
       opacity: 1,
+      stroke: "#252525",
```

Comment:
Oh... but with a default positive color of white you will have to add a bit of extra logic to get that to look correct.